templates/commands/render/action_include.go

```diff
@@ -126,3 +150,35 @@ func includePath(ctx context.Context, inc *spec.IncludePath, sp *stepParams) err
 	}
 	return nil
 }
+
+// TODO(#158): add support for ignoring file name. For example, "/foo" ignores
+// a single file named foo in the root, but "foo" ignores all files named foo
+// anywhere.
+// checkIgnore checks the given path against the given patterns, if given
+// patterns is not provided, a default list of patterns is used.
+func checkIgnore(patterns []model.String, path string) (bool, error) {
+	if len(patterns) > 0 {
+		for _, p := range patterns {
+			matched, err := filepath.Match(filepath.FromSlash(p.Val), path)
+			if err != nil {
+				return false,
+					p.Pos.Errorf("failed to match path (%q) with pattern (%q): %w", path, p.Val, err)
+			}
+			if matched {
+				return matched, nil
```

Comment:
Our convention would be to say `return true` if we already know that the value of `matched` is `true`.